packages/workflow/src/internals.ts

```diff
@@ -416,9 +425,56 @@ export class State {
     [
       '__enhanced_stack_trace',
       (): EnhancedStackTrace => {
-        // const stacks = this.getStackTraces();
-        // const sourceMap = this.sourceMap;
-        return { locations: [], sources: {} };
+        const sourceMap = this.sourceMap;
```

Comment:
I know I wrote this original line but you could replace with:
```suggestion
        const { sourceMap } = this;
```